### ISSUE TITLE
Integrate Auto Release Management

### ISSUE THREAD

**Issue by katielevy1** [intuit/truffle-shuffle#3]
Integrate Auto to support releases: https://github.com/intuit/auto

**Comment by nikhilbansal97** [comment]
Hey @katielevy1! I would like to work on this issue!

**Comment by katielevy1** [comment]
@nikhilbansal97 awesome, let me know if you have any questions!

**Comment by nikhilbansal97** [comment]
@katielevy1 Sure!

**Comment by su7edja** [comment]
@nikhilbansal97 are you still interested in working on this? Otherwise, I can do it.

**Comment by nikhilbansal97** [comment]
@su7edja Yes, I will be working on it.

**Comment by nikhilbansal97** [comment]
Hey @katielevy1 Whenever a PR is merged, should we create a release for the whole project or just the library? I see that `1.0.0` release contains the whole project.

**Comment by hwangange** [comment]
Hi Katie, I'm a GHC participant and would like to work on this issue!

**Comment by satrivaicci** [comment]
@katielevy1 Is this issue still open? Would like to help with this integration.

**Comment by sanchi0204** [comment]
@katielevy1  I want to work on this issue. Can you help me how to navigate through it?

**Comment by sanchi0204** [comment]
> @katielevy1 I want to work on this issue. Can you help me how to navigate through it?

@katielevy1 Can I take this ?

**Comment by katielevy1** [comment]
@sanchi0204 Yes, you can take a look at the onboarding docs here: https://intuit.github.io/auto/docs This issue might be a little more advanced from the other issues but a good challenge!
